src/move/econia/sources/registry.move

```diff
@@ -281,7 +281,7 @@ module econia::registry {
         market_id: u64,
         /// Number of base units exchanged per lot.
         lot_size: u64,
-        /// Number of quote units exchanged per tick.
+        // Number of quote units exchanged per tick.
```

Comment:
@qdrs I assume this was an accident? We don't want to be making any modifications to Move code.